server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -203,10 +206,26 @@ const virtualCardMutations = {
         UserId = user.id;
       }
 
-      return await virtualCard.update({
-        privateData: { ...(virtualCard.get('privateData') as Object), cvv, expireDate },
+      const updateAttributes = {
+        name: args.name,
         UserId,
-      });
+      };
+
+      if (args.monthlyLimit && virtualCard.spendingLimitInterval === 'MONTHLY' && virtualCard.provider === 'STRIPE') {
```

Comment:
Not a requested change.

It's generally a good practice (at least in our codebase) to establish constants for keywords such as 'MONTHLY' and 'STRIPE' .